enterprise/developerbox/installation/board-recovery.md

```diff
@@ -15,6 +20,65 @@ permalink: /documentation/enterprise/developerbox/installation/board-recovery.md
 
 ***
 
+# Serial flasher update
+
+This section is for the people who do not have UART mezzanine.
+Updating the serial flasher, you can use serial flasher without a UART mezzanine.
```

Comment:
It would be useful to describe in more detail the consequences of doing this. For example something like (replacing <....> the right names):

"By applying these changes then <...> will be replaced with an alternative version of <...> firmware that presents its user interface on the micro-B serial port instead of the Low Speed connector."